shift-react-app/src/steps.ts

```diff
@@ -0,0 +1,65 @@
+// tslint:disable:no-console
+import {
+  readFileSync,
+  writeFileSync,
+} from 'fs';
+import { join } from 'path';
+import findLastIndex from 'lodash.findlastindex';
+import { spawn } from 'child_process';
+
+export function setupProxy() {
+  const proxyFile = join('src', 'setupProxy.js');
+  const proxyCode = `const { setupProxy } = require('@binaris/shift-local-proxy');
+module.exports = setupProxy(__dirname);
+`;
+  writeFileSync(proxyFile, proxyCode, { flag: 'wx' });
+  console.log(`Created ${proxyFile}, please commit this file`);
+}
+
+export function installPackages() {
+  return new Promise((resolve, reject) => {
+    const dependencies = [
+      '@binaris/shift-local-proxy',
+      '@binaris/shift-babel-macro',
+      '@binaris/shift-fetch-runtime',
+      '@binaris/shift-db',
```

Comment:
Subscriptions are a part of the product, in #102 I didn't update the installer script since I knew you were working on this PR.
`@binaris/shift-backend-babel-plugin` is a required package.
I'm still not sure how a bundle will work since we need to distinguish between packages for frontend, backend and dev environment.

Reply:
let's discuss the list of the packages outside the scope of this PR. It currently reflects the original installer it replaces.
(In general, users should not get access to APIs which they will not have remote equivalent of)